doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -795,10 +795,10 @@
     alignas(T) element_t myAlignedArray[Capacity];
 
     // after
-    #include "iceoryx_hoofs/containers/uninitialized_array.hpp"
+    #include "iox/detail/uninitialized_array.hpp"
```

Comment:
Since this wasn't in `internal` before, would it make sense to keep it in `iox/uninitialized_array.hpp`?

We did not specify how to handle this in https://github.com/eclipse-iceoryx/iceoryx/issues/1593#issuecomment-1258266337 so I guess it would be fine. I think I also need to move the comment from that issue to a more prominent location

Reply:
Well, it was in `internal` before but the include path in the release notes have been wrong :grimacing: